client/src/app/app.component.ts

```diff
@@ -99,6 +100,8 @@ export class AppComponent implements OnInit, AfterViewInit {
     this.openModalsIfNeeded()
 
     this.document.documentElement.lang = getShortLocale(this.localeId)
+
+    setTimeout(cleanupVideoWatch, 1500) // Run in timeout to ensure we're not blocking the UI
```

Comment:
Move this in video-watch component to not import player stuff in the main component